File: VB/Dashboard_CustomSchemaProvider/Form1.vb
Namespace Dashboard_CustomSchemaProvider

    Public Partial Class Form1
        Inherits DevExpress.XtraEditors.XtraForm

        Public Sub New()
            InitializeComponent()
            dashboardDesigner1.CreateRibbon()
            dashboardDesigner1.LoadDashboard("Data\dashboard-with-products.xml")
            ' This propery is used to specify the database schema provider.
            ' Uncomment one of the following lines depending on the provider.
            dashboardDesigner1.CustomDBSchemaProviderEx = New ManualDBSchemaProvider()
        ' dashboardDesigner1.CustomDBSchemaProviderEx = new LimitDBSchemaProvider();
        End Sub
    End Class
End Namespace

File: CS/Dashboard_CustomSchemaProvider/Form1.cs
using DevExpress.DashboardWin;

namespace Dashboard_CustomSchemaProvider {
    public partial class Form1 : DevExpress.XtraEditors.XtraForm {
        public Form1() {
            InitializeComponent();
            dashboardDesigner1.CreateRibbon();
            dashboardDesigner1.LoadDashboard(@"Data\dashboard-with-products.xml");

            // This propery is used to specify the database schema provider.
            // Uncomment one of the following lines depending on the provider.
            dashboardDesigner1.CustomDBSchemaProviderEx = new ManualDBSchemaProvider();
            // dashboardDesigner1.CustomDBSchemaProviderEx = new LimitDBSchemaProvider();
        }
    }
}
